signingscript/tests/conftest.py

```diff
@@ -32,7 +32,7 @@ async def noop_async(*args, **kwargs):
     pass
 
 
-@pytest.yield_fixture(scope="function")
+@pytest.fixture(scope="function")
 def tmpfile():
     with tempfile.NamedTemporaryFile(delete=False) as tmp:
```

Comment:
Hm. We should probably keep `yield_fixture`, without `delete=False`, and `yield tmp.name`, so we clean up at the end. This doesn't make things worse, though.